cert/aziot-certd/src/http/get_or_import_or_delete.rs

```diff
@@ -73,6 +73,21 @@ impl http_common::server::Route for Route {
     type PostBody = serde::de::IgnoredAny;
     type PostResponse = ();
 
-    type PutBody = serde::de::IgnoredAny;
-    type PutResponse = ();
+    type PutBody = aziot_cert_common_http::import_cert::Request;
+    type PutResponse = aziot_cert_common_http::import_cert::Response;
+    async fn put(
+        self,
+        body: Self::PutBody,
+    ) -> http_common::server::RouteResponse<Self::PutResponse> {
+        let mut inner = self.inner.lock().await;
+        let inner = &mut *inner;
+
+        match inner.import_cert(&self.cert_id, &body.pem.0) {
+            Ok(()) => (),
+            Err(err) => return Err(super::to_http_error(&err)),
+        };
+
+        let res = aziot_cert_common_http::import_cert::Response { pem: body.pem };
```

Comment:
Is there any particular reason we're parroting back the pem?

Reply:
Just for consistency. It's quite common for "create an object with this spec" API to return the spec to you.